app/api/groups/get_children.go

```diff
@@ -32,6 +32,8 @@ type groupChildrenViewResponseRow struct {
 	IsOpen bool `json:"is_open"`
 	// required:true
 	IsPublic bool `json:"is_public"`
+	// Only returned if the current user is a manager of the group.
+	IsEmpty *bool `json:"is_empty,omitempty"`
```

Comment:
Maybe adding what it means. `Whether the group has no child group (incl. users)`

Reply:
fixed.